### PR TITLE
Expose kafka with nodeport

### PR DESCRIPTION
**Pull request by yanmxa**
fixes: https://github.com/stolostron/backlog/issues/23767
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: test/setup/kafka/kafka-cluster.yaml [hunk 1]
```diff
@@ -51,31 +54,3 @@ spec:
...
-metadata:
-  name: status
-  namespace: kafka
-  labels:
-    strimzi.io/cluster: kafka-brokers-cluster
-spec:
-  partitions: 1
-  replicas: 2
-  config:
-    cleanup.policy: compact
```
Comment (morvencao) [inline test/setup/kafka/kafka-cluster.yaml:81]: we don't need these two resources in testing any more?

Comment (yanmxa) [reply test/setup/kafka/kafka-cluster.yaml:81]: Because it's more about the hoh, not the Kafka itself, I move it to hub-of-hubs setup later.

Comment (yanmxa) [reply test/setup/kafka/kafka-cluster.yaml:81]: I discussed with @clyang82 , and we agreed that should keep Postgres and Kafka middleware as separate as possible. Such as the Kafka cluster instance. we wanted i to contain less information about HoH.

File: test/setup/kind/top-hub-config.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+kind: Cluster
```
Comment (clyang82) [inline test/setup/kind/top-hub-config.yaml:1]: suggest to use hoh-config.yaml

File: test/setup/kafka/kafka-cluster.yaml (hunk 1 above)
Comment (clyang82) [reply test/setup/kafka/kafka-cluster.yaml:81]: I am not quit sure why remove the topics. the topic is used to organize messages. HoH should have 2 topics.

Comment (yanmxa) [reply test/setup/kafka/kafka-cluster.yaml:81]: Just like we move the postgres-init.yaml to hoh_setup.sh. I move the topics which are specific for hoh to hoh_setup.sh too.

**Review by morvencao — COMMENTED** [review]
/LGTM

**Pull request merged**